src/Surfnet/StepupMiddlewareClientBundle/Configuration/Dto/InstitutionConfigurationOptions.php

```diff
@@ -34,6 +34,14 @@ class InstitutionConfigurationOptions implements Dto
     public $showRaaContactInformation;
 
     /**
+     * @Assert\Type(type="array", message="middleware_client.dto.configuration.allowed_second_factors.must_be_array")
+     * @Assert\All(
+     *     @Assert\Type(type="string", message="middleware_client.dto.configuration.allowed_second_factors.item_must_be_string")
+     * )
+     */
+    public $allowedSecondFactors;
```

Comment:
As you stated in the other PR, we are using these strings to intersect with the globally configured `enabled_second_factors`. As an aside: we do not have `SecondFactorType` (or the Stepup-bundle for that matter) in the MiddlewareClientBundle yet.